source/agora/network/NetworkManager.d

```diff
@@ -415,6 +423,14 @@ public class NetworkManager
         client.registerListener();
     }
 
+    public void startPeriodicNameRegistration ()
+    {
+        this.registry_client = this.getNameRegistryClient(
+            validator_config.registry_address, 2.seconds);
+        this.onRegisterName();  // avoid delay
+        this.taskman.setTimer(5.minutes, &this.onRegisterName, Periodic.Yes);
```

Comment:
So we send a registration every 5 minutes, no matter what ? That works, but it should be documented as a conscious decision.

Reply:
document it just alongside with the code?

Reply:
> So we send a registration every 5 minutes, no matter what ? That works, but it should be documented as a conscious decision.

there are multiple reasons why we want to register in every 5 mintues:
1. what if the registry server is restarted?
2. what if on the client machine a new ip was added -> we want to register that one too